demo/src/client/faro/initialize.ts

```diff
@@ -19,6 +20,9 @@ export function initializeFaro(): Faro {
       ...getWebInstrumentations({
         captureConsole: true,
       }),
+      new FetchInstrumentation({
+        ignoredUrls: [`http://localhost:${env.faro.portAppReceiver}/collect`],
```

Comment:
The default transport (Fetch) which is added by Faro [automatically add the collector URL defined above to the transport](https://github.com/grafana/faro-web-sdk/blob/f799029fad306b2a16c1a29cef068576f079ab23/packages/web-sdk/src/config/makeCoreConfig.ts#L31-L32).

So you can get the default collector url in the instrument like this:

```ts
  private getIgnoreUrls(): Array<string | RegExp> {
    return this.transports.transports?.flatMap((transport) => transport.getIgnoreUrls());
  }
  ```

Reply:
Very very clean, thank you @codecapitano